lib/utils/store.dart

```diff
@@ -41,9 +41,9 @@ class Store extends GetxController {
 
     if (response.statusCode == 200) {
       isConnectionError = false;
-      this.user.connect(username);
+      this.user.connect(user.username);
       this.channel =
-          IOWebSocketChannel.connect("ws://$ipAddr:41431/ws/$username");
+          IOWebSocketChannel.connect("ws://$ipAddr:41431/ws/$user.username");
```

Comment:
Since we need the value of `username`, it'll be `${user.username}` instead of `$user.username`.

Reply:
Hey @sanket143!
I have made the changes!
could you please review it?